Fix Identical Icons MDFloatingActionButtonSpeedDial

VICTORVICKIE opened the pull request:
### Description of Changes
Checked with KitchenSink and everything works as expected and it allows users/dev to use identical icons in speed dials

data provided should be as follows,

### Examples

```
data = {'Python':'language-python' ,
            'PHP':'language-php' ,
            'C++':'language-cpp' ,
           }
```
![Screenshot_3](https://user-images.githubusercontent.com/45727291/111903744-c5d1eb00-8a69-11eb-8e81-90704a90abf7.jpg)
```
data = {'Python':'plus' ,
            'PHP':'plus' ,
            'C++':'plus' ,
           }
```
![Screenshot_2](https://user-images.githubusercontent.com/45727291/111903701-91f6c580-8a69-11eb-82b0-b45b9d67958e.jpg)

VICTORVICKIE committed:
Merge pull request #1 from kivymd/master

Updating my fork

VICTORVICKIE committed:
Fix - same icon - MDFloatingActionButtonSpeedDial

VICTORVICKIE committed:
Update button.py

VICTORVICKIE committed:
Update button.py

VICTORVICKIE committed:
Update stack_buttons.py

VICTORVICKIE committed:
Update stack_buttons.py

HeaTTheatR commented:
@VICTORVICKIE 
Please write more informative commits. What exactly did you do and why.

![Снимок экрана 2021-03-21 в 17 12 53](https://user-images.githubusercontent.com/16930280/111908095-ee0d1a00-8a68-11eb-82ed-ef94f68a2df2.png)

VICTORVICKIE committed:
Added Identical Icon Speed dials 

Added Identical Icon Speed dials in MDFloatingActionButtonSpeedDial and 
updated docs for the data in MDFloatingActionButtonSpeedDial

VICTORVICKIE committed:
changed data for Identical icons speed dial format

changed data for Identical icons speed dial format in KitchenSink demo

VICTORVICKIE commented:
> @VICTORVICKIE
> Please write more informative commits. What exactly did you do and why.
> 
> ![Снимок экрана 2021-03-21 в 17 12 53](https://user-images.githubusercontent.com/16930280/111908095-ee0d1a00-8a68-11eb-82ed-ef94f68a2df2.png)

Sorry, now I've added informative commit message

HeaTTheatR commented on demos/kitchen_sink/libs/baseclass/stack_buttons.py:
```diff
@@ -15,6 +15,7 @@ def callback(instance_button):
 
             button_speed_dial = MDFloatingActionButtonSpeedDial()
             button_speed_dial.root_button_anim = True
+            # {"Text to be displayed" : "Icon name"}
```
Please remove these comments - they are not needed in this context

HeaTTheatR commented on kivymd/uix/button.py:
```diff
@@ -1872,6 +1872,9 @@ def on_data(self, instance, value):
         # Bottom buttons.
+        # Added New format for data of speed dial for identical icons
+        # {"Text to be displayed":"icon name"}
+        
```
Please remove these comments - they are not needed in this context

VICTORVICKIE committed:
changed data for Identical icons speed dial format

VICTORVICKIE committed:
Added Feature for identical speed dial icons

Added Feature for identical speed dial icons in MDFloatingActionButtonSpeedDial

HeaTTheatR commented:
@VICTORVICKIE 
Please, refactor your code - https://github.com/kivymd/KivyMD#setup-environment

![Снимок экрана 2021-03-21 в 18 16 46](https://user-images.githubusercontent.com/16930280/111910291-d5553200-8a71-11eb-80d7-9617700a874a.png)

VICTORVICKIE committed:
Added Identical icon for speed dials
